client/tw/categorical.ts

```diff
@@ -162,7 +146,7 @@ export class CatValues extends CategoricalBase {
 }
 
 export class CatPredefinedGS extends CategoricalBase {
-	//term: CategoricalTerm
+	// term, type, isAtomic, $id are set in ancestor base classes
 	q: PredefinedGroupSettingQ
 	#groupset: BaseGroupSet
```

Comment:
Why is `#groupset: BaseGroupSet` necessary? Term groupsetting should already be in `this.term`.

Reply:
setting a `groupset` instance property would allow abstraction of what groupset refers to, between predefined-groupset and custom-groupset. In this way, the same addon method can refer to just `this.groupset` without needing to test if it's a CatPredefinedGS or CatCustomGS. However, in order for an addon method to access the `groupset` property, it cannot be private (with a hash prefix), I'll need to change to a non-enumerable `groupset` property (I'll do it in this PR).